### PR TITLE
Workspace

### PR DESCRIPTION
**Pull request by mehmetSuzer**
Continue #588 

Fixed some overflow errors.
Names of the contributors are written line by line now by ListView.builder instead of using '\n'.
Latex is added to mobile and web (preview - write modes)
Entry creation is done by a widget similar to entry card instead of entry popup form. I had difficulty adding scrollable latex to it.
Pending contributors are visible in mobile now.
 CardContainer is now background-colorable.

Note: Edit entry is not working for now. I couldnt solve the problem.

### CODE REVIEW

File: project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/contributor_card.dart [hunk 1]
```diff
@@ -1,15 +1,19 @@
+import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:go_router/go_router.dart';
 import '../../../../models/user.dart';
+import '../../../../utils/colors.dart';
```
Comment (defabdullah) [inline project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/contributor_card.dart:5]: can you update this styling. ıt should be similar to package:colloborative_science_platform/...

File: project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/mobile_entry_card.dart [hunk 2]
```diff
@@ -0,0 +1,256 @@
+import 'dart:convert';
+
+import 'package:collaborative_science_platform/utils/responsive/responsive.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_tex/flutter_tex.dart';
+
+import '../../../../models/workspaces_page/entry.dart';
```
Comment (defabdullah) [inline project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/mobile_entry_card.dart:7]: similar to my previous comment

File: project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/new_entry.dart [hunk 3]
```diff
@@ -0,0 +1,228 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_tex/flutter_tex.dart';
+
+import '../../../../utils/colors.dart';
```
Comment (defabdullah) [inline project/FrontEnd/collaborative_science_platform/lib/screens/workspace_page/mobile_workspace_page/widget/new_entry.dart:6]: similar to previous ones

**Review by Zulalm — CHANGES_REQUESTED** [review]
Content of the entries is not shown before clicking write button.
image
Please remove the "name surname (email)" line since we decided to display them only in the node details page in the last lab meeting.

**Comment by mehmetSuzer** [comment]
@defabdullah @Zulalm  import statements are fixed and the latex renderer works in web as well now.

**Comment by mehmetSuzer** [comment]
Now, users can send a message while sending collaboration request.

**Comment by mehmetSuzer** [comment]
I added a semantic tag list view with a search bar to the workspaces. It has mock data for now. 
addSemanticTags function takes List<int> on the provider side, but the model of SemanticTags has "String" ids. There is problem that should be fixed.

**Review by Zulalm — APPROVED** [review]

**Pull request merged**